x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/XPackClientPlugin.java

```diff
@@ -411,12 +413,6 @@
             )
         );
 
-        // rollupV2
-        if (RollupV2.isEnabled()) {
```

Comment:
I removed the feature flag we had used for Rollups v2. Now, rollups code works as an operation for time_series indices

Reply:
Added the TSDB feature flag